src/System.Data.Common/src/System/DBNull.cs

```diff
@@ -1,31 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Yes, we need to.  We need typeof(DBNull).TypeCode to return TypeCode.DBNull, and we need Convert.IsDBNull(DBNull.Value) to return true, both of which require that the type identity of DBNull match what's in System.Private.CoreLib.